CONTRIBUTORS.md

```diff
@@ -2,7 +2,7 @@
 
 Add yourself as a contributor to this project as a Markdown link that links your name to your GitHub profile and shows your favorite emoji, as in the following example:
 
-    - [Dione Developer](https://github.com/DioneDeveloper) 💅
+    - ✨ [Dione Developer](https://github.com/DioneDeveloper) 💅
```

Comment:
I mentioned this open-source software called SourceTree a while back and it has really helped me when files get added to my commits where I didn't mean for things to get added -- here is a good example, if this kind of stuff comes up a lot and you want to see line by line what is actually getting added to your commits highly recommend a visual git tool like SourceTree altho there are many others

Reply:
Oh this is so interesting! I'm curious about why this gets added?